tests/nc-server/simple_server.c

```diff
@@ -228,7 +228,7 @@ drop(struct conn_list_node * node_ptr)
 	if (node_ptr->network_read_cookie != NULL)
 		network_read_cancel(node_ptr->network_read_cookie);
 
-	/* Close the incoming connection. */       
+	/* Close the incoming connection. */
```

Comment:
I like the change, but does this really belong in a PR called "libcperciva import"?